helpers/prism.js

```diff
@@ -28,6 +28,7 @@ codeLanguages.set('plain', 'Plain Text');
 codeLanguages.set('python', 'Python');
 codeLanguages.set('r', 'R');
 codeLanguages.set('racket', 'Racket');
+codeLanguages.set('regex', 'Regex');
```

Comment:
Mm, did we formally decide what we were going to do about languages CM doesn't support? I know for examples like Racket, the thought was "Lisp is probably good enough", but for sure there's no equivalent like that for regex.

Reply:
Yeah, plain text is certainly fine. I know there was some concern about trying to prioritize consistency over breadth. I don't personally care and I think supporting more stuff in Prism than we do in CM seems fine, but... I can't remember if that's where we actually landed.

Reply:
Circling back to this after discussion with designers, if CodeMirror does not support the language, we will format it as plain text in CodeMirror, while Prism will apply the expected formatting. We will not exclude those languages from the list.  If CodeMirror has reasonable formatting based on a different language, I think we should try to use it (ex. Arduino -> C++/C/C-like).

Reply:
Yeah, seems good to me!